src/main/java/net/fabricmc/loom/decompilers/fernflower/AbstractFernFlowerDecompiler.java

```diff
@@ -65,12 +65,12 @@ public void decompile(Path compiledJar, Path sourcesDestination, Path linemapDes
 
 		project.getLogging().captureStandardOutput(LogLevel.LIFECYCLE);
 
-		Map<String, Object> options = new HashMap<String, Object>() {{
+		Map<String, Object> options = new HashMap<>() {{
 				put(IFernflowerPreferences.DECOMPILE_GENERIC_SIGNATURES, "1");
 				put(IFernflowerPreferences.BYTECODE_SOURCE_MAPPING, "1");
 				put(IFernflowerPreferences.REMOVE_SYNTHETIC, "1");
 				put(IFernflowerPreferences.LOG_LEVEL, "trace");
-				put(IFernflowerPreferences.THREADS, metaData.numberOfThreads);
+				put(IFernflowerPreferences.THREADS, metaData.numberOfThreads());
 			}};
```

Comment:
Hmm, the code format on the last line here looks a bit odd too

Reply:
Yeah, this is checkstyle's doing. Might leave it for now